framework/platform/parsers/CLI11.cpp

```diff
@@ -172,4 +153,83 @@ std::vector<std::string> CLI11CommandParser::get_command_value(Command *command)
 
 	return it->second->results();
 }
+
+// TODO: Explain this
```

Comment:
Also need to update the new command-line pattern in the documentation (see https://github.com/KhronosGroup/Vulkan-Samples#usage)